.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[eos-panel-extension.all]
+source_file = locale/eos-panel-extension.pot
```

Comment:
Based on the other patches which put the translations in 'po/' this cannot be right, surely:

```suggestion
source_file = po/eos-panel-extension.pot
```

Reply:
Fixed, bad copy paste.